include/clap/helpers/host.hxx

```diff
@@ -428,15 +428,15 @@ namespace clap { namespace helpers {
    ///////////////
    template <MisbehaviourHandler h, CheckingLevel l>
    Host<h, l> &Host<h, l>::from(const clap_host *host) noexcept {
-      if constexpr (l >= CheckingLevel::Minimal) {
```

Comment:
ouch, maybe it is time to require C++17? ;-)
It is 2024 so it is already 7 years old.

Reply:
Yes, I would be fine with that
Maybe @baconpaul can tell? My original idea was to use `clap::helpers::Host` for clap-wrapper as well. So it should meet the requirements there, too. I think somewhere I read that they wrote something like "c++17 min" but just want to ensure that it would be OK.

The CMakeLists.txt defaulted to 11 before we started upgrading it recently and still does I think. So I guess we would have to change it there as well (?)

Reply:
Ok just removed the CLAP_IF_CONSTEXPR
We can solve the rest (c++11 etc) later